index.bs

```diff
@@ -632,19 +632,281 @@ Issue(12): Propose control protocol for Remote
----------------------------------------------
+[[PRESENTATION-API#security-and-privacy-considerations]] place these
+requirements on the Open Screen Protocol:
+
+1.  Presentation URLs and presentation IDs should remain private among the
+    parties that are allowed to connect to a presentation, per the
+    cross-origin access guidelines.
+1.  Controllers and recievers should be notified when multiple connections have
+    been made to a presentation, per the user interface guidelines.
+1.  Messaging between presentations and controllers should be authenticated and
+    confidential, per the guidelines for messaging between presentation
+    connections.
+
+The Open Screen Protocol addresses these considerations by:
+
+1. Requiring mutual authentication and a TLS-secured QUIC connection before
+     presentation URLs, IDs, or messages are exchanged.
+1. Adding explicit messages and connection IDs for individual
+     {{PresentationConnection|PresentationConnections}} so that controllers and
+     receivers can track the number of active connections.
+
+Remote Playback API Considerations {#remote-playback-considerations}
+----------------------------------
+
+The [[REMOTE-PLAYBACK#security-and-privacy-considerations]] also state that
+messaging between local and remote playback devices should also be authenticated
+and confidential.
+
+This consideration is handled by requiring mutual authentication and a
+TLS-secured QUIC connection before any remote playback related messages are
+exchanged.
+
+Mitigation Strategies {#security-mitigations}
+--------------------------------------------
+
+### Local passive network attackers ### {#local-passive-mitigations}
+
+Local passive attackers may attempt to harvest data about user activities and
+device capabilties using the Open Screen Protocol.  The main strategy to address
+this is data minimization, by only exposing opaque public key fingerprints
+before user-mediated authentication takes place.
+
+Passive attackers may also attempt timing attacks to learn cryptographic the
+cryptographic parameters of the TLS 1.3 QUIC connection.
+
+Issue(130): [Security] Review attack and mitigation considerations for TLS 1.3
+
+### Local active network attackers ### {#local-active-mitigations}
+
+Local active attackers may attempt to impersonate a presentation display the
+user would normally trust.  The [[#authentication]] step of the Open Screen
+Protocol prevents a man-in-the-middle from impersonating a controller or
+receiver, without knowledge of a shared secret.  However, it is possible for an
+attacker to impersonate an existing, trusted display or a newly discovered
+display that is not yet authenticated and try to convince the user to
+authenticate it.
+
+This can be addressed through a combination of techniques.  The first is flagging:
+
+* Flag advertised displays whose friendly name, public key fingerprint, or
+    IP:port collide with an already trusted display.
```

Comment:
I think we should flag when we see the friendly name with a different fingerprint, but not if we see the same IP:port with a different friendly name or FP (an AP may reuse IPs and client may choose ports fairly statically).  And I don't think we should flag the same FP but different IP:port (since IP is likely to change).  The only other combination seems to be same FP but different friendly name, which should also be flagged.

Reply:
I broke this into two criteria:
- Flag FP collision.
- Flag new friendly name for a previously trusted FP.
I think that covers the cases you describe.